CleverTapSDK/CleverTap.m

```diff
@@ -1135,6 +1138,14 @@ - (BOOL)isMuted {
 
 - (void)applicationDidBecomeActive:(NSNotification *)notification {
     [self _appEnteredForeground];
+    
+#if !CLEVERTAP_NO_INAPP_SUPPORT
+    // Update push permission status everytime app is resumed.
+    self.pushPrimerManager.pushPermissionStatus = CTPushNotKnown;
```

Comment:
This line should be removed. We are changing the status when the app is backgrounded, so once it becomes active, the status should already be `CTPushNotKnown`.

Reply:
Done